.github/workflows/build.yml

```
@@ -0,0 +1,252 @@
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Get Version from package.json
+        id: get_version
+        run: |
+          version="v$(jq -r '.version' package.json)"
+          echo "::set-output name=version::$version"
+      - name: Check if Tag Exists
+        id: check_tag
+        run: |
```

actionlint (1.7.7)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)